app/modules/video-players/jwplayer.js

```diff
@@ -85,6 +85,8 @@ export default class JWPlayer extends BasePlayer {
 			},
 			this.params.onCreate.bind(this, bidParams)
 		);
+
+		window.M.loadScript(Ads.getInstance().jwPlayerMoat.pluginURL, true);
```

Comment:
Can we call `adsInstance.jwPlayerMoat.loadTrackingPlugin()` instead?
I'd suggest having only one loader of this plugin.